controllers/discordactions.js

```diff
@@ -86,6 +85,19 @@ const getAllGroupRoles = async (req, res) => {
   }
 };
 
+const getGroupsRoleId = async (req, res) => {
+  try {
+    const { discordId } = req.userData;
+    const userRolesArray = await discordRolesModel.getGroupRolesForUser(discordId);
```

Comment:
I think userGroupRoles or just userRoles would be a good name. Having array in the name is not required.

Reply:
Done changes has been done